app/src/main/java/com/brewfind/Service/BreweryService.kt

```diff
@@ -9,6 +9,11 @@ import retrofit2.Callback
 import retrofit2.Response
 
 class BreweryService {
+
+    fun fetchBreweriesByName(breweryName: String) : MutableLiveData<ArrayList<Brewery>>{
+        return MutableLiveData<ArrayList<Brewery>>()
+    }
```

Comment:
Added a new method which would call and return the data in the Brewery array list

Reply:
This is incomplete functionality, as it's not actually fetching breweries by name.  It's fetching an empty MutableLiveData object.